src/Controller/EnMarche/Jecoute/AbstractJecouteController.php

```diff
@@ -69,6 +70,11 @@ public function jecouteSurveyCreateAction(
         SuggestedQuestionRepository $suggestedQuestionRepository,
         UserInterface $user
     ): Response {
+        if (JecouteSpaceEnum::CANDIDATE_SPACE === $this->getSpaceName()
```

Comment:
what do you think about creating a function `checkCreateAccess` and use it in this generic createAction so we can override it in `JecouteCandidateController` for doing this specific job?
```
$this->checkCreateAccess();
```

Reply:
@illusionOfParadise, could you override the `checkCreateAccess` in `JecouteCandidateController`, please?